logistic_requisition_donation/i18n/fr.po

```diff
@@ -0,0 +1,43 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* logistic_requisition_donation
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-01-29 15:40+0000\n"
+"PO-Revision-Date: 2015-01-29 15:40+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: logistic_requisition_donation
+#: model:ir.model,name:logistic_requisition_donation.model_logistic_requisition_cost_estimate
+msgid "Create cost estimate of logistic requisition lines"
+msgstr "Create cost estimate of logistic requisition lines"
```

Comment:
@lepistone The issue there is that those strings will be seen as translated but they aren't. Am I wrong?